lib/active_merchant/billing/gateways/payscout.rb

```diff
@@ -102,7 +102,7 @@ def add_creditcard(post, creditcard)
       end
 
       def parse(body)
-        Hash[body.split('&').map { |x| x.split('=') }]
+        body.split('&').map { |x| x.split('=') }.to_h
```

Comment:
This one will fail if there is no value in any of the keys if `body='foo=bar&buzz=bang&hello='` it will throw an argument error. So I think we should keep it as is, and find the rule and either add an exception or turn it off for this file.

Reply:
Okay, thank you. I'll update it